src/nvim/message.c

```diff
@@ -302,9 +302,13 @@ void msg_multihl(HlMessage hl_msg, const char *kind, bool history)
   msg_ext_set_kind(kind);
   for (uint32_t i = 0; i < kv_size(hl_msg); i++) {
     HlMessageChunk chunk = kv_A(hl_msg, i);
-    msg_multiline(chunk.text, chunk.hl_id, true, false, &need_clear);
+    if (err) {
+      emsg_multiline(chunk.text.data, chunk.hl_id, true);
```

Comment:
This makes me think the `kind` should be "emsg" ?

Reply:
IDK; see my comment above and `eval.c:7972` for the `:echoerr` behavior. We are trying to mirror that here.

Reply:
ok I see that :echoerr uses the same call: https://github.com/neovim/neovim/blob/17b46d01e29443452ae8b607017f8f5c585d3f0a/src/nvim/eval.c#L7968-L7972